src/legacy/status_im/data_store/communities.cljs

```diff
@@ -25,7 +25,7 @@
                       (case k
                         "tokenGated"              :token-gated?
                         "canPost"                 :can-post?
-                        "can-view"                :can-view?
+                        "canView"                 :can-view?
                         "hideIfPermissionsNotMet" :hide-if-permissions-not-met?
```

Comment:
the `can-view` key never appeared, we were handling this prop wrongly, maybe due to an update on the status-go response?